changelogs/fragments/puppet_debugging_options.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - Add support for --debug, --verbose, --summarize
```

Comment:
The general format for changelog entries is `- [module name] - [what changed] ([link to GitHub issue])`. [Example](https://github.com/ansible/ansible/blob/devel/changelogs/fragments/lineinfile-empty-regexp.yml)

Reply:
Valid sections can be found in the [reno config.yaml](https://github.com/ansible/ansible/blob/133b70ff4503be52b007d3abe642d7d7f360b686/changelogs/config.yaml).

Reply:
Where would I have found this in the contributors/community guide? Did I miss it or is it just having to know where in the code to look?  I was expecting find something ala:
https://docs.openstack.org/reno/latest/user/index.html

Reply:
We recently switched to `reno` and haven't documented it well yet. 😞